vulkan/vulkan_native_surface_android.cc

```diff
@@ -4,6 +4,8 @@
 
 #include "flutter/vulkan/vulkan_native_surface_android.h"
 
+#include <android/native_window.h>
```

Comment:
Did you mean to stage this? This doesn't seem related to the vulkan stuff.

Reply:
The old Vulkan headers were including `android/native_window.h`, but the new headers are not